### DISCUSSION TITLE
How secure is it for integrating data provider?

### DISCUSSION THREAD

**Discussion by ralphilius** [refinedev/refine#6509] [General]
Hello,

I want to know if the credentials for data provider would be secure because I see one of the deploy option is static export. 

Regards,

**Comment by BatuhanW** [comment]
Hello @ralphilius can you explain a bit more? Hard to understand from the question. Best.

**Reply by ralphilius** [reply]
I mean I have Rest API with write access. Will refine be secure for exposing it?

**Reply by BatuhanW** [reply]
I don't think this has something to do with Refine. We are a frontend framework, so if your Rest API is secure for exposing to any frontend, answer is the same for Refine too.
